webrtc.html

```diff
@@ -523,39 +529,45 @@
           </li>
         </ol>
 
-        <p>Once the RTCPeerConnection object has been initialized, for every
-        call to <code>createOffer</code>, <code>setLocalDescription</code>,
-        <code>createAnswer</code>, <code>setRemoteDescription</code>,
-        and <code>addIceCandidate</code>,
-        execute the following steps:</p>
+        <p>An <code><a>RTCPeerConnection</a></code> object has an
+        <dfn>operations queue</dfn>, [[<a>operations</a>]], which enforces that
+        only one operation in the queue is executed at a given time. If
```

Comment:
Not a must, but how about: "only one asynchronous operation in the queue is executed concurrently." ?